packages/editor/templates/Coin/README.md

```diff
@@ -52,24 +52,25 @@ The `mint` function sends an amount of newly created coins to another address. T
 
 Anyone can use the `send` (who already has some of these coins) to send coins to anyone else. If the sender does not have enough coins to send, the `require` call fails and provides the sender with an appropriate error message string.
 
-### Configure
+### 1. Configure
 
 ![Configure the contract](./images/hw-configure.png)
-Clicking the disclosure triangle next to the contract name reveals three sub-sections. In the _Configure_ sub-section you can change the name of the contract and set the initial values sent to the constructor arguments. In this example, the string displayed in the front end interface.
 
-### Compile
+Configuring the contract allows you to set the name of the contract and the initial values sent to the constructor as arguments. You can configure the contract by clicking on the gear icon in the left panel.
+
+### 2. Compile
 
 ![Compile the contract](./images/hw-compile.png)
 
-Solidity is a compiled language, and you need to convert the Solidity code into bytecode before the contract can run. Clicking the disclosure triangle next to the contract name reveals three sub-sections, you can compile the contract by clicking the _Compile_ sub-section, and output appears in the _Output_ pane.
+Solidity is a compiled language, and you need to convert the Solidity code into bytecode before the contract can run. In the file tree under the contract file, you can compile the contract by clicking the _Compile_ sub-section, and output will appear in the _Output_ pane.
```

Comment:
```suggestion
Solidity is a compiled language, and you need to convert the Solidity code into bytecode before the contract can run. In the file tree under the contract file, you can compile the contract by clicking the _Compile_ sub-section, and output appears in the _Output_ pane.
```

Gonna be fussy and rewrite the inappropriate future tense, not just because I'm fussy, but mostly because I didn't use it anywhere else ;p

I hope…